psydac/api/essential_bc.py

```diff
@@ -43,6 +43,28 @@ def check_boundary_type(bc):
 
 #==============================================================================
 def apply_essential_bc_stencil(a, *, axis, ext, order, identity=False):
+    """ This function applies the homogeneous boundary condition to the Stencil objects,
+        by setting the boundary degrees of freedom to zero in the StencilVector,
+        and the corresponding rows in the StencilMatrix/StencilInterfaceMatrix to zeros.
+        If the identity keyword argument is set to True, the boundary diagonal terms are set to 1.
+
+    Parameters
+    ----------
+    a : LinearOperator/Vector
+        The LinearOperator/Vector that will be modified.
+
+    axis : int
+        Axis of the boundary.
+
+    ext : int
+        Extremity of the boundary.
```

Comment:
It would be useful to say that the only values allowed are -1 and 1.

Reply:
done